sound/soc/sof/sof-priv.h

```diff
@@ -26,6 +26,7 @@
 #define SOF_DBG_TEXT		BIT(3)
 #define SOF_DBG_PCI		BIT(4)
 #define SOF_DBG_RETAIN_CTX	BIT(5)	/* prevent DSP D3 on FW exception */
+#define SOF_DBG_LOG_ERROR	BIT(6)
```

Comment:
I find the name rather confusing. The rest are related to very specific cases, here 'log_error' doesn't tell me when it's to be used.

Reply:
would adding a comment like "/* used to dump dsp status with error log level */" help?

Reply:
SOF_DBG_FORCE_ERR_LEVEL?